pkg/runtime/wfengine/wfengine.go

```diff
@@ -46,37 +42,50 @@ type WorkflowEngine struct {
 	actorRuntime   actors.Actors
 	startMutex     sync.Mutex
 	disconnectChan chan any
+	config         *WFConfig
 }
 
 var (
 	wfLogger            = logger.NewLogger("dapr.runtime.wfengine")
 	errExecutionAborted = errors.New("execution aborted")
 )
 
+// WFConfig is the configuration for the workflow engine
+type WFConfig struct {
+	AppID             string
+	WorkflowActorType string
+	ActivityActorType string
+}
+
 func IsWorkflowRequest(path string) bool {
 	return backend.IsDurableTaskGrpcRequest(path)
 }
 
-func NewWorkflowEngine() *WorkflowEngine {
+func NewWorkflowEngine(config *WFConfig) *WorkflowEngine {
+	config.WorkflowActorType = actors.InternalActorTypePrefix + utils.GetNamespaceOrDefault() + "." + config.AppID + ".workflow"
+	config.ActivityActorType = actors.InternalActorTypePrefix + utils.GetNamespaceOrDefault() + "." + config.AppID + ".activity"
 	// In order to lazily start the engine (i.e. when it is invoked
 	// by the application when it registers workflows / activities or by
 	// an API call to interact with the engine) we need to inject the engine
 	// into the backend because the backend is what is registered with the gRPC
 	// service and needs to have a reference in order to start it.
 	engine := &WorkflowEngine{}
+	engine.config = config
 	be := NewActorBackend(engine)
 	engine.backend = be
-	engine.activityActor = NewActivityActor(be)
-	engine.workflowActor = NewWorkflowActor(be)
+	engine.activityActor = NewActivityActor(be, config)
+	engine.workflowActor = NewWorkflowActor(be, config)
 
 	return engine
 }
 
 // InternalActors returns a map of internal actors that are used to implement workflows
 func (wfe *WorkflowEngine) InternalActors() map[string]actors.InternalActor {
 	internalActors := make(map[string]actors.InternalActor)
-	internalActors[WorkflowActorType] = wfe.workflowActor
-	internalActors[ActivityActorType] = wfe.activityActor
+	if wfe.config != nil {
```

Comment:
when can the config here be nil? 
Please add UTs for the same as necessary.

Reply:
Ideally, config here won't be null as per the way current code is written, but I have placed a check accordingly.

Reply:
Added UT.